tests/test_tasks/test_task.py

```diff
@@ -1,85 +1,66 @@
-import sys
-
-if sys.version_info[0] >= 3:
-    from unittest import mock
-else:
-    import mock
-
-from time import time
-import numpy as np
+import unittest
 
 import openml
 from openml.testing import TestBase
+from openml.datasets import OpenMLDataset
+from openml.tasks import OpenMLTask
+from openml.exceptions import OpenMLServerException
+from .test_supervised_task import OpenMLSupervisedTaskTest
 
-
+# Helper class
+# The test methods in this class
+# are not supposed to be executed.
 class OpenMLTaskTest(TestBase):
-    _multiprocess_can_split_ = True
+    # task id, dataset_id,
+    # estimation_procedure
+    # will be set from the
+    # extending classes
+
+    def setUp(self):
+
+        super(OpenMLTaskTest, self).setUp()
+        self.task_id = 11
+        self.estimation_procedure = 23
+
+    @classmethod
+    def setUpClass(cls):
+
+        if cls is OpenMLTaskTest or OpenMLSupervisedTaskTest:
+            raise unittest.SkipTest(
+                "Skip OpenMLTaskTest tests,"
+                " it's a base class"
+            )
+        super(OpenMLTaskTest, cls).setUpClass()
+
+    def test_download_task(self) -> OpenMLTask:
 
-    @mock.patch('openml.tasks.functions.get_dataset', autospec=True)
```

Comment:
yes, I think we discussed this and we agreed that we would remove it since we are actually checking the get_dataset function. Or do I remember wrong?

Reply:
Unlikely, the reason I am asking is because removing this test also removes the only check that the class labels are loaded. Could you please add such a check again (obviously only for the classification and learning curve task)? Also, it would be good if that test actually used a task from the server instead of a task which is kept locally.